roboticslogger/file_handler.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import errno
+import datetime
+
+def get_timestamped_filepath():
+    """ Gets a timestamped filepath for the logfiles """
+    return "{0}/Logs/RoboticsLogger/".format(os.path.expanduser('~')) 
```

Comment:
To be safe, you might want to use something that the library provides to build fs filepaths. For example to achieve the above you'd do this in ruby:

~~~~ruby
path = File.join("path", "to", "file")
# "path/to/file"
~~~~

By delegating that to some library, or the std library of whatever language you're using, you're able to not worry about cross platform problems. (A nice reading would be the rust [PathBuf][1]... please don't kill me).

[1]: https://doc.rust-lang.org/std/path/struct.PathBuf.html